.gitignore

```diff
@@ -29,4 +29,10 @@ $RECYCLE.BIN/
 # Visual Studio
 .vscode/
 .history/
-*.code-workspace
\ No newline at end of file
+*.code-workspace
+
+<<<<<<< HEAD
```

Comment:
It looks like this is a merge conflict, that will need to be resolved between one branch (between <<<<<<< HEAD and ======= and between ======= and >>>>>>> add-images).
Since this is the `.gitignore` file (the file that tells git what to ignore, we probably want to keep both of the lines that are being added (`.breakpoints` and `public/`).